projects/storefrontlib/src/cms-components/product/product-tabs/product-details-tab/product-details-tab.component.html

```diff
@@ -1,3 +1,20 @@
 <ng-container *ngIf="product$ | async as product">
-  <div class="container" [innerHTML]="product?.description"></div>
+  <div class="container">
+    <div
+      *ngIf="!showMore; else fullContent"
+      [innerHTML]="product?.description.substring(0, 400)"
+    ></div>
+    <ng-template #fullContent
```

Comment:
We can use `product.description.substr(0, 400)` no matter the description length longer than 400 or not, right?

Reply:
Yes.